planning/behavior_path_planner/src/util/lane_change/util.cpp

```diff
@@ -107,6 +107,18 @@ std::vector<int64_t> replaceWithSortedIds(
   }
   return original_lane_ids;
 }
+
+double getLateralDistance(
+  const lanelet::ConstLanelets & lanelet_sequence, const geometry_msgs::msg::Pose & pose)
+{
+  lanelet::ConstLanelet closest_lanelet;
+  lanelet::utils::query::getClosestLanelet(lanelet_sequence, pose, &closest_lanelet);
+  const auto & centerline_2d = lanelet::utils::to2D(closest_lanelet.centerline());
+
+  const auto lanelet_point = lanelet::utils::conversion::toLaneletPoint(pose.position);
+  return lanelet::geometry::signedDistance(
+    centerline_2d, lanelet::utils::to2D(lanelet_point).basicPoint());
+}
```

Comment:
Can this be moved in `utilities`? It doesn't seem to be lane change specific, so other module might benefit from this function as well.

Reply:
@zulfaqar-azmi-t4 done